cypress/plugins/nodeManager/plugin.ts

```diff
@@ -108,25 +111,34 @@ class Node {
     this.httpPort = options.id;
     this.peerPort = options.id;
     this.proxyBinaryPath = path.join(ROOT_PATH, options.proxyBinaryPath);
+    this.storagePath = path.join(CYPRESS_WORKSPACE_PATH, uuid.v4());
   }
 
   async start() {
     this.logger.log("starting node");
 
+    fs.mkdirsSync(this.storagePath);
+
+    const envCopy: NodeJS.ProcessEnv = {};
+    Object.assign(envCopy, global.process.env);
+    envCopy["RAD_HOME"] = this.storagePath;
+
     const process = childProcess.spawn(
       this.proxyBinaryPath,
       [
         "--test",
+        "--test-use-rad-home",
         "--http-listen",
         `${HOST}:${this.httpPort}`,
         "--peer-listen",
         `${HOST}:${this.peerPort}`,
       ],
-      {}
+      { env: envCopy }
```

Comment:
There’s no need to copy all environment variables. The child process inherits all variables that are not explicitly set.
```suggestion
      { env: { RAD_HOME: this.storagePath } }
```

Reply:
I tried this out, it doesn't work for some reason.

```
const envCopy: NodeJS.ProcessEnv = {};
Object.assign(envCopy, global.process.env);
envCopy["RAD_HOME"] = this.storagePath;

const process = childProcess.spawn("env", [], {
  env: envCopy,
});
```
Results in the full env being shown:

```
STDOUT: npm_package_author_email=dev@radicle.xyz
npm_package_dependencies_ethers=^5.0.23
npm_package_devDependencies__types_sinon=^9.0.10
npm_package_scripts_test_unit_watch=jest --watchAll
npm_package_devDependencies__types_twemoji=^12.1.1
npm_config_version_git_sign=
npm_config_ignore_scripts=
npm_config_user_agent=yarn/1.22.5 npm/? node/v14.5.0 darwin x64
npm
...
```

But:
```
const process = childProcess.spawn("env", [], {
  env: { RAD_HOME: "mystoragepath" },
});
```

Results in only the one variable being set:
```
 STDOUT: RAD_HOME=mystoragepath
```

Reply:
Yes you’re right. A more idiomatic and concise way to write this would be
```suggestion
      { env: { ...process.env, RAD_HOME: this.storage_path } }
```

Reply:
Good call!

5ec74caf75c4e1e0896f7e4d3123c8e594da991a